drivers/scheduler/anthos/anthos.go

```diff
@@ -200,6 +201,13 @@ func (anth *anthos) Init(schedOpts scheduler.InitOptions) error {
 	if err := anth.getVersion(); err != nil {
 		return err
 	}
+	anth.skipAdminClusterUpgrade = true
```

Comment:
instead of `skipAdminClusterUpgrade` if you define `adminClusterUpgrade` and then you would not need this line
```
anth.skipAdminClusterUpgrade = true
```

Then you can set anth.adminClusterUpgrade = true on line: 207

Reply:
Good catch. Incorporated this comment.